cookbook/case_studies/feature_engineering/sqlite_datacleaning/README.rst

```diff
@@ -0,0 +1,90 @@
+Data Cleaning
+-------------
+Feature Engineering off-late has become one of the most prominent topics in Machine Learning. 
+It is the process of transforming raw data into features that better represent the underlying problem to the predictive models, resulting in improved model accuracy on unseen data.
+
+This tutorial will implement data cleaning of SQLite3 data, which does both data imputation and univariate feature selection. These are so-called feature engineering techniques.
+
+Why SQLite3?
+============
+SQLite3 is written such that the task doesn't depend on the user's image. It basically:
+
+- Shifts the burden of writing the Dockerfile from the user using the task in workflows, to the author of the task type
+- Allows the author to optimize the image that the task runs
+- Works locally and remotely
+
+.. note::
```

Comment:
This is the same for sqlalchemy. As for Athena, big query, hive etc, a container is not required. The queries are registered with FlyteAdmin and send directly to the respective engines